chatbot_ner/setup_logs.py

```diff
@@ -0,0 +1,114 @@
+import collections
+from typing import Dict, Any, NamedTuple, List
+
+import logging
+import structlog
+import django.dispatch
+from django.http import HttpRequest
+from django_structlog.middlewares.request import get_request_header
+from django_structlog.signals import bind_extra_request_metadata
+
+
+class _LoggingKey(NamedTuple):
+    namespace: str
+    bind_key: str
+    header_key: str
+    meta_key: str
+
+
+class LoggingKeys(object):
```

Comment:
Since this is a class, please add a docstring to indicate its purpse.

Reply:
Another question here, will logging fail if these keys are not present? We should log without these keys if they're not. This is an open source repository and somebody else might be using it somewhere. Don't want to break their flow.

Reply:
In cases these keys are not present it will not create any error.
So in case this keys are find in headers they will be logged else ignored